src/Sulu/Bundle/Sales/OrderBundle/Resources/views/Pdf/order.dynamically.html.twig

```diff
@@ -0,0 +1,5 @@
+{% extends templateDynamicBasePath %}
+
+{% block pdf_title %}
+    Warenkorb
```

Comment:
t extends a template that has everything else hardcoded in german language... Therefore I didnt found it useful to make this the only translated word there... And it is just the fallback template.